dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -62,13 +64,63 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         }
     }
 
+    void PushNotificationManager::RegisterUnpackagedApplicationHelper(
+        const winrt::guid& remoteId,
+        _Out_ wil::unique_cotaskmem_string &unpackagedAppUserModelId)
+    {
+        wchar_t processName[1024];
+        THROW_HR_IF(ERROR_FILE_NOT_FOUND, GetModuleFileNameExW(GetCurrentProcess(), NULL, processName, sizeof(processName) / sizeof(processName[0])) == 0);
+
+        THROW_IF_FAILED(::CoInitializeEx(nullptr, COINITBASE_MULTITHREADED));
+
+        auto scopeExit = wil::scope_exit(
+            [&]() { CoUninitialize(); });
+
+        wil::com_ptr<INotificationsLongRunningPlatform> notificationPlatform{
```

Comment:
Since you named the interface in the template parameter, you can just say `auto notificationPlatform { ... }` here